.env.tmp

```diff
@@ -54,11 +54,13 @@ CONTAINER_TAG=""
 #
 
 CONFIG_OUTPUT_DIR="config"
-DEPLOYMENT_CONFIG_FILE="deployment.json"
 DEPLOYMENT_CONFIG_TEMPLATE_FILE="deployment.template.json"
-LOGS_PATH="logs"
+DEPLOYMENT_CONFIG_DEBUG_TEMPLATE_FILE="deployment.debug.template.json"
```

Comment:
I think this file should be named `deployment.template.debug.json`.

Reply:
We would like to allow users pass deployment manifest templates with any name, but we also want to set some rules so that we can determine what the generated deployment manifest would be named. The current rule is that if the template is XXX.template.json, the generated one will be named config/XXX.json. So when deployment.debug.template.json is built, the generated file would be config/deployment.debug.json` (the naming is implemented at https://github.com/Azure/iotedgedev/pull/358/files#diff-4a4d85e50c4c4f0af9d1ca17c4252649R166)